README.md

```diff
@@ -130,7 +130,46 @@ Note: during D day. There is a lot of work to be done. The lesser people there a
 
 ### Emcee
 
+#### Expectations
+
+The emcees help transition between sections of the conference. They also engage the audience with additional insight and help facilitate the Q&A. They also provide time for the speaker to get ready. 
+
+#### People
+
+At any point in time, there must be an emcee, and there should be a backup emcee. It is highly recommended to rotate emcee roles each day or each half-day. 
+
+#### Tasks
 
+##### Pre-Conference
+
+- Prepare speaker introduction
+  - Correct pronunciation of speaker names
+- Prepare talk introduction
+- If necessary: Talk and speaker introduction slide to show the audience. Work with the designer.
+
+Emcees should make preparations for themselves. Their own material can come in any form, like slides, notes or cards. [BYOD](https://en.wikipedia.org/wiki/Bring_your_own_device). Verify that there is space to locate your laptop (or in some venues an additional screen).
+
```

Comment:
Some tips Kheng Meng mentioned before:
Prepare some material to fill some time gaps when necessary.
For eg: when waiting for speakers to setup/arrive, 
The materials he prepared were in the form of fact bites on the events, venue, or speakers.
He also prepares a bunch of trivial questions which he can engage the audience while filling the time gaps.